mslib/msui/topview.py

```diff
@@ -318,7 +318,7 @@ def changeMapSection(self, index=0, only_kwargs=False):
 
         # Create a keyword arguments dictionary for basemap that contains
         # the projection parameters.
-        kwargs = current_map["map"]
+        kwargs = dict(current_map["map"])
         kwargs.update({"CRS": current_map["CRS"], "BBOX_UNITS": proj_params["bbox"],
```

Comment:
@joernu76 I'm thinking about how to write a test for this. We need a new dict when we don't want to "update" the origin too.

```
>>> a = {'a': {'b':1}}
>>> c = a['a']
>>> c.update({'v': 4}
>>> c
{'b': 1, 'v': 4}
>>> a
{'a': {'b': 1, 'v': 4}}
```